Add Table of Contents and reorganize CONTRIBUTING.md for better navigation (#547)
Azure/azure-mcp: CONTRIBUTING.md

    - `LiveTests/` - Live tests depend on Azure resources and authentication
- `docs/` - Documentation

## Development Workflow

### Development Process

1. Fork the repository
2. Create a feature branch
3. Make your changes
4. Write or update tests
5. Test locally
6. Submit a pull request

### Adding a New Command

1. **Create an issue** with title: "Add command: azmcp [namespace] [resource] [operation]" and detailed description

2. **Set up development environment**:
   - Open VS Code Insiders
   - Open the Copilot Chat view
   - Select "Agent" mode

> 3. **Generate the command** using Copilot:

Reviewer: ```suggestion
3. **Generate the command using Copilot**:
```

    - `LiveTests/` - Live tests depend on Azure resources and authentication
- `docs/` - Documentation

## Development Workflow

### Development Process

1. Fork the repository
2. Create a feature branch
3. Make your changes
4. Write or update tests
5. Test locally
6. Submit a pull request

### Adding a New Command

> 1. **Create an issue** with title: "Add command: azmcp [service] [resource] [operation]" and detailed description

Reviewer 2: The "service" is also renamed to "namespace"

2. **Set up development environment**:
   - Open VS Code Insiders
   - Open the Copilot Chat view
   - Select "Agent" mode

3. **Generate the command** using Copilot:
   ```
   Execute in Copilot Chat:

>    "create [service] [resource] [operation] command using #new-command.md as a reference"

Reviewer 2: the "service" is renamed to "namespace"

   - Open the Copilot Chat view
   - Select "Agent" mode

> 3. **Generate the command using Copilot**:

Reviewer: Revert this change

Instructions for consuming the package from the dev feed can be found in the "Extensions" tab of the pipeline run page.

## Support and Community

### Questions and Support

We're building this in the open.  Your feedback is much appreciated, and will help us shape the future of the Azure MCP server.

👉 [Open an issue in the public repository](https://github.com/Azure/azure-mcp/issues/new/choose).

### Additional Resources

- [Azure MCP Documentation](https://github.com/Azure/azure-mcp/blob/main/README.md)
- [Command Implementation Guide](https://github.com/Azure/azure-mcp/blob/main/src/Docs/new-command.md)
- [VS Code Insiders Download](https://code.visualstudio.com/insiders/)
- [GitHub Copilot Documentation](https://docs.github.com/en/copilot)

### Code of Conduct

> This project adheres to a standard code of conduct. By participating, you are expected to uphold this code.

Reviewer: ```suggestion
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments. By participating, you are expected to uphold this code.
```

### License

> By contributing, you agree that your contributions will be licensed under the project's license.

Reviewer: ```suggestion
By contributing, you agree that your contributions will be licensed under the project's [license](https://github.com/Azure/azure-mcp/blob/main/LICENSE).
```
